src/app/test_executive/test_executive.ml

```diff
@@ -85,8 +94,9 @@ let report_test_errors error_set
           Bash_colors.none ) ;
     Out_channel.(flush stderr) ) ;
   (* TODO: re-enable error check after libp2p logs are cleaned up *)
-  (* if num_errors > 0 || num_missing_events > 0 then exit 1 else Deferred.unit *)
-  if num_missing_events > 0 then exit 1 else Deferred.unit
+  (* if num_errors_total > 0 || num_missing_events > 0 then exit 1 else Deferred.unit *)
```

Comment:
should the comment here be updated to mention `num_errors_internal`?

Reply:
i would imagine that when the time comes to comment that code back in, we would continue using num_errors_total because that already includes num_errors_internal